### PR TITLE
Update final_video.py

### PR DESCRIPTION
**Pull request by Hassan-Hossam**
# Description

Bug fix

# Issue Fixes

This fixed corrupt background files which lead to an error where moviepy can't read frames

# Checklist:

- [ ] I am pushing changes to the **develop** branch
- [ ] I am using the recommended development environment
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have formatted and linted my code using python-black and pylint
- [ ] I have cleaned up unnecessary files
- [ ] My changes generate no new warnings
- [ ] My changes follow the existing code-style
- [ ] My changes are relevant to the project

### CODE REVIEW

File: video_creation/final_video.py [hunk 1]
```diff
@@ -303,7 +310,7 @@ def make_final_video(
     text = f"Background by {background_config[2]}"
     background_clip = ffmpeg.drawtext(
         background_clip,
-        text=text,
+        text=(''),
```
Comment (OpenSourceSimon) [inline video_creation/final_video.py:313]: This will remove the whole watermark, we're not gonna do that

File: video_creation/final_video.py [hunk 2]
```diff
@@ -99,8 +99,15 @@ def prepare_background(reddit_id: str, W: int, H: int) -> str:
...
         output.run(quiet=True)
     except Exception as e:
-        print(e)
-        exit()
+        print('An error occurred while running...')
+        print('trying auto error fixing...')
+        try:
+            os.remove(f"assets/temp/{reddit_id}/background.mp4")
+            os.remove(f"assets/temp/{reddit_id}/background_noaudio.mp4")
+            prepare_background(reddit_id, W, H)
```
Comment (OpenSourceSimon) [inline video_creation/final_video.py:107]: This won't work, you first need to get the background video from a different file

File: video_creation/final_video.py [hunk 3]
```diff
@@ -99,8 +99,15 @@ def prepare_background(reddit_id: str, W: int, H: int) -> str:
     try:
         output.run(quiet=True)
     except Exception as e:
-        print(e)
-        exit()
+        print('An error occurred while running...')
+        print('trying auto error fixing...')
```
Comment (OpenSourceSimon) [inline video_creation/final_video.py:103]:
```suggestion
```

File: video_creation/final_video.py [hunk 4]
```diff
@@ -99,8 +99,15 @@ def prepare_background(reddit_id: str, W: int, H: int) -> str:
...
+        try:
+            os.remove(f"assets/temp/{reddit_id}/background.mp4")
+            os.remove(f"assets/temp/{reddit_id}/background_noaudio.mp4")
+            prepare_background(reddit_id, W, H)
+        except:
+            print('Please Rerun the script ')
```
Comment (OpenSourceSimon) [inline video_creation/final_video.py:109]:
```suggestion
            print('Please re-run the script ')
```

**Review by OpenSourceSimon — CHANGES_REQUESTED** [review]

File: video_creation/final_video.py (hunk 2 above)
Comment (Hassan-Hossam) [reply video_creation/final_video.py:107]: it does work tho

Comment (OpenSourceSimon) [reply video_creation/final_video.py:106]: You first need to call `chop_background_video()`, otherwise there is no background

Comment (Hassan-Hossam) [reply video_creation/final_video.py:106]: one sec going to add it

Comment (Hassan-Hossam) [reply video_creation/final_video.py:106]: so i did some test and figured out that it works without the chop_background_video() try it out for yourself

**Pull request closed without merging**